<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Facturas</class>
 <widget class="QDialog" name="Facturas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre_:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hora:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_fecha">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_hora">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Saldo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saldp">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>360</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>precio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_impuesto">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>390</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>impuesto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>descuento:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_descuento">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>420</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TOTAL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>total</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancelar">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PAGAR</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_factura">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>531</width>
     <height>181</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Lineas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombres</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>precio</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>cantidad</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
